api/resolvers/notifications.js

```diff
@@ -168,6 +168,19 @@ export default {
           LIMIT ${LIMIT}+$3)`
       )
 
+      // territory transfers
+      if (meFull.noteTerritoryTransfers) {
+        queries.push(
+          `(SELECT "TerritoryTransfer".id::text, "TerritoryTransfer"."created_at" AS "sortTime", NULL as "earnedSats",
+            'TerritoryTransfer' AS type
+            FROM "TerritoryTransfer"
+            WHERE "TerritoryTransfer"."newUserId" = $1
+            AND "TerritoryTransfer"."created_at" <= $2
+            ORDER BY "sortTime" DESC
+            LIMIT ${LIMIT}+$3)`
+        )
+      }
+
```

Comment:
I don't think we need settings for notice of territory transfers:

1. I can't imagine they happen often
2. Founders need to know that it happens when it does